searx/settings.yml

```diff
@@ -196,6 +196,10 @@ engines:
 #  - name : base
 #    engine : base
 #    shortcut : bs
```

Comment:
Nit: Could you please add one more empty line above this?

Reply:
> Nit: Could you please add one more empty line above this?

Oops .. done!